.github/workflows/check.yaml

```diff
@@ -0,0 +1,62 @@
+name: Check
+
+on:
+  pull_request:
+    branches: [ main ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  check:
+    runs-on: [self-hosted, kstest]
+
+    steps:
+      - name: Checkout change
+        uses: actions/checkout@v2
+        with:
+          path: permian
+
+      - name: Clone repository with additional plugins and tests
+        run: |
+          git  -c http.sslVerify=false clone --depth 1 ${{ secrets.ADDITIONAL_TESTS_REPO }} permian-redhat
```

Comment:
Shouldn't the variable be called something like secrets.ADDITIONAL_PLUGINS_REPO ?

Reply:
Yes, that is better name for it